app/controllers/point_details_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PointDetailsController < ApplicationController
-  before_action :require_resort_or_group_leader
+  before_action :require_resort_or_group_leader_or_rvt_member
```

Comment:
Why `require_application_or_evaluation_season` is absent in this file? If I understand this correctly, resort leaders will be able to edit in offseason, which is not a huge flaw, but it would be nicer if we  `require_application_or_evaluation_season`, I think.